code/hdb/startup.q

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Maybe a little unnecessary, but can we have a comment on WHY this new code is needed?